### PR TITLE
Update post_training_quantization.md

### PR DESCRIPTION
**Pull request by mathemage**
Fix Markdown formatting for the bullet points of use cases

### CODE REVIEW

**Comment by mathemage** [comment]
I've just signed the CLA:

image

**Comment by gbaned** [comment]
> I've just signed the CLA:
> 
> image
> 
> @gbaned Can you update my permissions, please? Thanks.

@mathemage  Can you please comment like this  **@googlebot I consent**  ? Thanks!

**Comment by mathemage** [comment]
@googlebot I consent

**Comment by mathemage** [comment]
> @googlebot I consent

@gbaned Have I done this correctly? Is there any other issue or blocker for this?

**Comment by mathemage** [comment]
> @googlebot I consent

@miaout17  Have I done this correctly? Is there any other issue or blocker for this?

**Comment by mathemage** [comment]
@googlebot I consent

**Comment by gbaned** [comment]
@mathemage Sorry for the late response. CLA looks good now. This PR is waiting for review.  Thanks!

**Comment by mathemage** [comment]
Hi @gbaned - any new progress?

Anything from my side to do?

**Comment by gbaned** [comment]
> Hi @gbaned - any new progress?
> 
> Anything from my side to do?

@mathemage Sorry for the delay. This PR is waiting for the review. Nothing pending from your end.  Thank you.

**Review by miaout17 — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

**Comment by mathemage** [comment]
Hi @gbaned ! Is there any problem with CLA from my side?

Let me know if you need something for me to do. Thanks!

**Comment by gbaned** [comment]
> Hi @gbaned ! Is there any problem with CLA from my side?
> 
> Let me know if you need something for me to do. Thanks!

Hi @mathemage  CLA looks good, nothing pending from your side. This PR is processing internally. Thank you!

**Review by gbaned — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

File: tensorflow/lite/g3doc/performance/post_training_quantization.md [hunk 1]
```diff
@@ -232,9 +232,11 @@ tflite_quant_model = converter.convert()
 </pre>
 
 Examples of the use cases where accuracy improvements provided by this
-quantization scheme include: * super-resolution, * audio signal processing such
-as noise cancelling and beamforming, * image de-noising, * HDR reconstruction
-from a single image.
+quantization scheme include: 
+*   super-resolution,
```
Comment (mihaimaruseac) [inline tensorflow/lite/g3doc/performance/post_training_quantization.md:236]:
```suggestion

*   super-resolution, 
```

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**